src/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Checkbox from './checkbox';
+
+export default {
+  component: Checkbox,
+  title: 'Components/Checkbox',
+  parameters: {
+    layout: 'fullscreen',
+  },
+} as ComponentMeta<typeof Checkbox>;
+
+const Template: ComponentStory<typeof Checkbox> = (args) => (
+  <Checkbox {...args} />
+);
+export const Regular = Template.bind({});
```

Comment:
Rather nitpicky, but important: Please add a new line between these two declarations.
```suggestion
);

export const Regular = Template.bind({});
```